pkg/kubelet/container/runtime.go

```diff
@@ -176,6 +176,8 @@ type RunContainerOptions struct {
 	// into docker's container runtime.
 	NetMode string
 	IpcMode string
+	// The parent cgroup to pass to Docker
+	CgroupRoot string
```

Comment:
Sorry to be a pain :) this one should be CgroupParent since at this point (in the runtime) it is the cgroup parent. Went with CgroupRoot for the flag since it is the "root cgroup under which Kubelet creates containers"